src/Pages/Liked/Liked.css

```diff
@@ -0,0 +1,6 @@
+.liked-container {
+  display: flex;
+  flex-direction: row;
```

Comment:
`flex-direction: row` is the default property of flex so no need to write it. Remove it and it will still work the same